src/styles/HamburgerStyle.ts

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
```

Comment:
I don't think this file is used anymore since Hamburger isn't. Can you double check and delete if its not?